salt/modules/mac_softwareupdate.py

```diff
@@ -26,15 +28,19 @@ def __virtual__():
     return __virtualname__
 
 
-def _get_upgradable(rec=False, restart=False):
+def _get_available(recommended=False, restart=False):
     '''
-    Utility function to get all upgradable packages.
+    Utility function to get all available update packages.
 
     Sample return date:
     { 'updatename': '1.2.3-45', ... }
     '''
-    cmd = 'softwareupdate --list'
-    out = __salt__['cmd.run_stdout'](cmd)
+    cmd = ['softwareupdate', '--list']
+    try:
+        out = salt.utils.mac_utils.execute_return_result(cmd)
+    except CommandExecutionError as exc:
+        raise CommandExecutionError(exc)
```

Comment:
As already discussed in #31959, this is very unusual. Could you please explain your reasoning here?